packages/core-js/utils/ast/finders.js

```diff
@@ -159,11 +126,48 @@ function _findFunctionSelectors(contractNode) {
   return selectors;
 }
 
+/**
+ * Get the complete tree of dependencies from the given contract. This methods
+ * takes an objects with the keys from all the contracts and the values are their
+ * AST nodes.
+ * @param {string} contractName
+ * @param {import("solidity-ast").SourceUnit[]} astNodes
+ * @returns {import("solidity-ast").ContractDefinition[]}
+ */
+function findContractDependencies(contractName, astNodes) {
```

Comment:
Changed the interface to require an Array of `astNodes`, as  for  the finder didn't make much sense the way  que store the modules using a key with the name.